TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+# Transaction Author Agreement V2
+```
+                             `-::-`                      
+                      ```   `//////`   ```               
+                    .////:` `//////` `:////.             
+                    //////-  `-::-`  -//////             
+                    .:////`    --    `////:.             
+                ....   ` .:`   --   `:. `   ....         
+              `/////:     `:.  ..  .:`     :////:`       
+              .//////-.`    .  ``  .    `.-//////.       
+               .:::````.---  -////-  ---.```::::.        
+                            .//////.                     
+               .-::-` `.---  -////-  ---.` `-::-.        
+              .//////-.`    `  ``  .    `.-//////.       
+              `/////:`    `:.  ..  .:`    `://///`       
+                .--.     .:`   --   `:.     .--.         
+                    .:////`    --    `////:.             
+                    //////-  `.::.`  -//////             
+                    .////:` `//////` `:////.             
+                      ```   `//////`   ```               
+                             `-::-`                      
+                                                                    
+ ssSSSSSss                                         II               
+sS"     "Ss                                        ""               
+SS,                                                                 
+`SSsssss,     ,ooOOOoo,   Vv       vV  Rr,rRRRrr,  II  Nn,nNNNnn,   
+  `"""""Ss,  oO"     "Oo  `Vv     vV'  RRR'   "RR  II  NNN'   `"Nn  
+        `Ss  OO       OO   `Vv   vv'   RR          II  NN       NN  
+SSs     SSS  "Oo,   ,oO"    `Vv,vV'    RR          II  NN       NN  
+ "SSSSSSS"    `"OoooO"'       "V"      RR          II  NN       NN  
+```
+[https://sovrin.org/](https://sovrin.org/)
+## **Summary:**
+* This summary is provided to help you understand your obligations when writing to the Sovrin Ledger Networks-it does not have any legal effect or replace the full legal text of the agreement provided below it.
```

Comment:
Consider removing this as a bullet list item and having it be a plain paragraph. That would cause the following bullets to have less indenting and matches a little better the intent of this section. Unless this is following the original text's layout; in which case, disregard my comment :slightly_smiling_face:

Reply:
this is better, thanks.